### PR TITLE
Support for video embedding

### PR DESCRIPTION
**Pull request by DLu**
### Description

Based on feedback in #546, I spun the video embedding out into this PR, which
 * ~~Copies the source code from [sphinx-contrib/video](https://github.com/sphinx-contrib/video/) directly. I was unable to figure out the proper way to install a dependency on the upstream extension.~~
 * Moves the video file in question to the static folder, ~~because the above extension doesn't seem to copy it to the destination folder like the image extension does.~~
 * Updates the embedding _by directly embedding the HTML required for the video to play_

~Its not the most elegant solution, but it works. I welcome feedback on how to make it more elegant. ~

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Comment by felixvd** [comment]
Thanks a lot for putting in the effort! I love that the video seems to work, and I don't mind the file living in `static` necessarily, but copied code needs to be maintained, so I'm wary about that.

I'm not familiar with the build process, so I'm not sure what you mean by `install a dependency on the upstream extension`. Wouldn't it work if you add the install instructions for this extension to [this script](https://github.com/ros-planning/moveit_tutorials/blob/master/htmlproofer.sh)?

**Comment by DLu** [comment]
I am just learning my way around sphinx. There seems like there should be a way to link to the released extension simply by adding `sphinxcontrib.video` in the `conf.py`, but that requires that the extension is [on the path somewhere](https://www.sphinx-doc.org/en/master/usage/extensions/index.html) and I'm unclear what the sphinx way to do that is. Add a `git clone` to the html_proofer` script?

**Comment by felixvd** [comment]
I don't know myself, but you can probably try what works in a `master-source` [docker container](https://github.com/ros-planning/moveit_tutorials/blob/master/.travis.yml#L23).

**Comment by AndyZe** [comment]
I'll just mention, one of the tutorials already has an embedded Youtube video.

https://ros-planning.github.io/moveit_tutorials/doc/realtime_servo/realtime_servo_tutorial.html

File: _scripts/video.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copied From https://raw.githubusercontent.com/sphinx-contrib/video/master/sphinxcontrib/video.py
```
Comment (davetcoleman) [inline _scripts/video.py:4]: Add more context about why this is copied:

> I was unable to figure out the proper way to install a dependency on the upstream extension.

File: doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst [hunk 2]
```diff
@@ -134,8 +134,10 @@ Moving Joints or in Null Space
 ++++++++++++++++++++++++++++++
 You can use the **Joints** tab to move single joints and the redundant joints of 7-DOF robots. Try moving the "null space exploration" slider as shown in the animation below.
 
-.. image:: rviz_joints_nullspace.webm
+.. video:: ../../_static/rviz_joints_nullspace.webm
```
Comment (davetcoleman) [inline doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst:137]: to @AndyZe 's point, youtube has just been embedded with HTML. wouldn't that be a lot easier than a custom extension?

```
.. raw:: html

    <object width="700" height="400">
      <embed
        src="https://www.youtube.com/v/8sOucNloJeI&hl=en_US&fs=1&rel=0"
        type="application/x-shockwave-flash" allowscriptaccess="always"
        allowfullscreen="true" width="700"
        height="385">
      </embed>
    </object>
```

**Comment by DLu** [comment]
I've updated this so that the video loops forever like a gif would. 

When I built it locally using the `./build_locally.sh` script, the generated html is 
```
<p>You can use the <strong>Joints</strong> tab to move single joints and the redundant joints of 7-DOF robots. Try moving the “null space exploration” slider as shown in the animation below.</p>
<video width="700px" nocontrols="true" autoplay="true" loop="true">
<source src="../../_static/rviz_joints_nullspace.webm" type="video/webm">
Illustration
</video>
```

**Review by felixvd — APPROVED** [review]
The directory/location might not technically be the cleanest way to write it, but if it works I'm happy with it.

**Comment by DLu** [comment]
I believe CI would pass for this now that #556 is merged.

**Comment by rhaschke** [comment]
Closing and reopening to trigger CI.

**Review by davetcoleman — APPROVED** [review]
Seems like this has been greatly simplified, for better or worse :-)

**Comment by davetcoleman** [comment]
@DLu this is probably worth adding to the repo README on best practice for embedding video/gif, so we don't go through this again

**Pull request merged**